contracts/src/components.cairo

```diff
@@ -11,4 +11,5 @@ mod quantity;
 mod metadata;
 mod caravan;
 mod trade;
-mod road;
\ No newline at end of file
+mod road;
+mod structure;
```

Comment:
i think the correct term for these is hyperstructures, can you change that everywhere in the code ?

Reply:
https://ebrary.net/127150/management/visions_city_future
After searching in chat history, i believe this is the inspiration for hyperstructures if you want to read a bit about it